packages/chain-controller/src/SnapChainProviderClient.test.ts

```diff
@@ -0,0 +1,79 @@
+import { ChainRpcMethod } from '@metamask/chain-api';
+import type { SnapId } from '@metamask/snaps-sdk';
+import { HandlerType } from '@metamask/snaps-utils';
+import type { Json } from '@metamask/utils';
+
+import { SnapChainProviderClient } from './SnapChainProviderClient';
+import { SnapHandlerClient } from './SnapHandlerClient';
+
+const snapId = 'local:localhost:3000' as SnapId;
+
+/**
+ * Builds a Snap chain API request.
+ *
+ * @param options0 - Chain API request object.
+ * @param options0.method - Chain API method to be called.
+ * @param options0.params - Chain API parameters.
```

Comment:
Is this a typo? `options0`?

Reply:
ah that was a suggestion my editor I guess, because I didn't give a name to the object I guess..

Do you want me to change that?

Reply:
I went for `request` instead, thanks!